packages/uniforms/src/SimpleSchemaBridge.js

```diff
@@ -202,7 +201,8 @@ export default class SimpleSchemaBridge extends Bridge {
             } else {
                 field = {
                     ...field,
-                    transform: value => options.find(option => option.value === value).label,
+                    transform: value => options.find(option => option.value === value) ?
+                        options.find(option => option.value === value).label : value,
```

Comment:
It's not a bug, but by design - if your field has allowed values _(indirectly, through `option`)_, then it's required to have current value there too. We can discuss it anyway, but please do it in a separate issue and PR.